src/measure_theory/group/prod.lean

```diff
@@ -89,7 +89,8 @@ variables [has_measurable_inv G]
 
 /-- The function we are mapping along is `S⁻¹` in [Halmos, §59],
   where `S` is the map in `map_prod_mul_eq`. -/
-@[to_additive map_prod_neg_add_eq]
+@[to_additive map_prod_neg_add_eq "The function we are mapping along is `-S` in [Halmos, §59], where
+`S` is the map in `map_prod_mul_eq`."]
```

Comment:
```suggestion
`S` is the map in `map_prod_sum_eq`."]
```
(looks like the additive name of `map_prod_mul_eq` is `map_prod_sum_eq`, as defined on line 60. That's possibly a mistake and should be `map_prod_add_eq`?)

Reply:
Renamed to `map_prod_add_eq` everywhere. Thanks!